### ISSUE TITLE
This repository should not be used anymore?

### ISSUE THREAD

**Issue by 1-21gigasats** [epiccurious/jade-diy#78]
Is there a reason why this repository is still open for new users while be have https://github.com/bitcoin-tools/diyjade where all the development happens? I think nobody should use this one here anymore as it currently misses the important fix of the disabled debugging mode.

I understand that this repo is more popular and easier to find buy maybe you can change the readme to point new users to the new repo or is there any other reason why this one is still available with no hint to the new repo?

**Comment by epiccurious** [comment]
You're right, this repo needs to be updated and archived with a link to the new repo.

I'll leave the script files in to avoid breaking anyone's workflow and update the readme+description.

**Comment by suttonle24** [comment]
Hey @epiccurious , I posted https://github.com/epiccurious/jade-diy/pull/86 to address this in the README from a messaging standpoint. Checking the open PRs, though, I do see that you still have https://github.com/epiccurious/jade-diy/pull/79 open since May which would essentially do the same thing.
Hope you might be able to merge one of those. This tool is awesome. Once I stopped burning time over here and got over to the latest repository (https://github.com/bitcoin-tools/diyjade) it worked on the first attempt. Much appreciated!
